tasks4.js

```diff
@@ -45,3 +45,42 @@ function sortSquareArray(int) {
 }
 
 console.log(sortSquareArray(integers));
+
+let competitions = [
+  ["HTML", "C#"],
+  ["C#", "Python"],
+  ["Python", "HTML"],
+];
+
+let results = [0, 0, 1];
+
+function findWinner(comp, res) {
+  let hashMap = new Map();
+  let currentPoints = 0;
+  let currentLeader = "";
```

Comment:
instead of 4 vars try to do it with 2 var's by using object. 
Object will hold score and winner team name